grakn-graql/src/test/java/ai/grakn/graql/internal/reasoner/ReasoningTests.java

```diff
@@ -655,6 +655,76 @@ public void transRelationWithNeqPredicate(){
         assertTrue(answers2.containsAll(answers));
     }
 
+    @Test
+    public void transConjunctionWithNeqPredicate(){
```

Comment:
Could you please provide more detailed test names or comment what these tests actually check?